app/views/studies/_expression_file_info_fields.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="row expression-file-info-fields">
+  <div class="col-sm-2">
+    <%= f.label :is_raw_counts, 'Is this a raw counts matrix?' %><br />
+    <%= f.radio_button :is_raw_counts, 1, checked: f.object.is_raw_counts, class: 'raw-counts-radio'  %>
+    <%= f.label :is_raw_counts, 'Yes', class: 'radio-pad' %>
+    <%= f.radio_button :is_raw_counts, 0, checked: !f.object.is_raw_counts, class: 'raw-counts-radio' %>
+    <%= f.label :is_raw_counts, 'No', class: 'radio-pad'  %>
+  </div>
+  <div class="col-sm-2">
+    <%= f.label :units %><br />
+    <%= f.select :units, options_for_select(ExpressionFileInfo::UNITS_VALUES, f.object.units),
+                 {include_blank: true},
+                 class: 'form-control raw-counts-select counts-unit-dropdown', disabled: !f.object.is_raw_counts %>
+  </div>
+  <div class="col-sm-3">
+    <%= f.label :biosample_input_type, "Biosample Input Type <i class='text-danger'>*</i>".html_safe %><br />
+    <%= f.select :biosample_input_type, options_for_select(ExpressionFileInfo::BIOSAMPLE_INPUT_TYPE_VALUES, f.object.biosample_input_type),
+                 {}, class: 'form-control' %>
```

Comment:
Without a prompt value, this will force the user to make a selection.  Since `biosample_input_type` is not required we need to allow blanks at both the UI and model level.
```suggestion
                 {prompt: 'Select one...'}, class: 'form-control' %>
```